store/tikv/snapshot.go

```diff
@@ -339,6 +339,20 @@ func (s *tikvSnapshot) batchGetSingleRegion(bo *Backoffer, batch batchKeys, coll
 				locks = append(locks, lock)
 			}
 		}
+		if batchGetResp.ExecDetailsV2 != nil {
+			if s.mu.stats != nil {
+				s.mu.Lock()
+				if s.mu.stats.scanDetail == nil {
+					s.mu.stats.scanDetail = &execdetails.ScanDetail{}
+				}
+				if s.mu.stats.timeDetail == nil {
+					s.mu.stats.timeDetail = &execdetails.TimeDetail{}
+				}
+				s.mu.stats.scanDetail.MergeFromScanDetailV2(batchGetResp.ExecDetailsV2.ScanDetailV2)
+				s.mu.stats.timeDetail.MergeFromTimeDetail(batchGetResp.ExecDetailsV2.TimeDetail)
+				s.mu.Unlock()
+			}
+		}
```

Comment:
```suggestion
		s.mergeExecDetail(batchGetResp.ExecDetailsV2)
```

and add below method:

```go
func (s *tikvSnapshot) mergeExecDetail(detail *pb.ExecDetailsV2) {
	s.mu.Lock()
	defer s.mu.Unlock()
	if detail == nil || s.mu.stats == nil {
		return
	}
	if s.mu.stats.scanDetail == nil {
		s.mu.stats.scanDetail = &execdetails.ScanDetail{}
	}
	if s.mu.stats.timeDetail == nil {
		s.mu.stats.timeDetail = &execdetails.TimeDetail{}
	}
	s.mu.stats.scanDetail.MergeFromScanDetailV2(detail.ScanDetailV2)
	s.mu.stats.timeDetail.MergeFromTimeDetail(detail.TimeDetail)
}
```